qualification/report/generate_pdf.py

```diff
@@ -875,11 +876,17 @@ def main():
     parser.add_argument("input", help="Output of pytest --report-log=...")
     parser.add_argument("pdf", help="PDF file to write")
     parser.add_argument("-c", "--commit-id", action="store_true", help="Output commit ID of katgpucbf image")
+    parser.add_argument(
+        "--report-docid",
```

Comment:
```suggestion
        "--report-doc-id",
```
For consistency with `--commit-id` (and it being treated as two words in variable names).